JRIngram/leitner
# Update BE tests to use in-memory server

# Pull Request (PR) - *Update BE tests to use in-memory server *
## Description of Change
- Uses a mongo-in-memory database for tests.
- Changes the DB_URL env var to be MONGO_URL and removes DB_PORT variable


## How to test this change
Run Unit tests and see that no changes on the front-end have occurred. i.e. data in place before running the tests is still in place after running the tests.
